src/app/veilederverktoy/historikk/components/oppgavehistorikk.tsx

```diff
@@ -13,9 +13,7 @@ function OppgaveHistorikkKomponent({ oppgaveHistorikk }: OwnProps) {
     const { oppgaveTema, oppgaveType } = oppgaveHistorikk;
     return (
         <div className="historikk__elem blokk-xs">
-            <Element>
-                <FormattedMessage id="innstillinger.modal.historikk-gosys-oppgave" />
-            </Element>
+            <Element>Gosys-oppgave opprettet</Element>
             <Normaltekst>
                 <FormattedMessage
```

Comment:
formattedMessage

Reply:
lar denne stå da oppgavehistorikk er satt opp på en annen måte enn resten som jeg ikke tør å røre i frykt for å ødelegge